### ISSUE TITLE
New attribute request: opposite of data-fred-class

### ISSUE THREAD

**Issue by digitalpenguin** [modxcms/fred#234]
Would it be possible to get a new Fred attribute that functions the opposite way to `data-fred-class`?
Currently `data-fred-class` adds the value as a class when Fred is not active.
What's needed is to be able to add a class only when Fred is active. 

This would be incredibly useful for styling dropzones when Fred is active, yet the styling would be invisible when a visitor views the page.

It would be kind of similar to `data-fred-render` only it would just be a class and not a whole div/element.

**Comment by theboxer** [comment]
I think you can use the `fred--block_content` class which is around every element when Fred is active.

**Comment by digitalpenguin** [comment]
Thanks! I can work with that.
This is definitely low priority but would still be great to have the attribute in a later version.

**Comment by rthrash** [comment]
You probably can also use `data-fred-block-class` if you need something Extra. From the to-be-published updated docs for the 1.1 release:

## data-fred-block-class

Allows you to specify an additional class which is added to the wrapping `div.fred--block` added around Elements when Fred is loaded. This can be useful when JavaScript libraries or CSS styling techniques require specific wrapper classes. When Fred is not loaded, the attribute will be added to the classes of the element itself.

### Example

Element Markup:
```html
<div class="image" data-fred-block-class="special-wrapper"></div>
```
When Fred is loaded, the markup around it will look as follows:

```html
<div class="fred--block special-wrapper">
    <div class="fred--toolbar">…</div>
    <div class="fred--block_content" data-fred-element-id="5ce33419-44d6-4e30-90db-8c9a62d04763" data-fred-element-title="Image">
        <div class="image"></div>
    </div>
</div>
```
When Fred is _not_ loaded, the processed markup for the element will look as follows:

```html
<div class="image special-wrapper"></div>
```

---

Using a `.special-wrapper .block-content` CSS selector would give you something more specific and unique, and even be able to do different things on different parts of the page as needed.

**Comment by digitalpenguin** [comment]
Thanks for the info - interesting!
